reportwidgets/browsers/partials/_widget.htm

```diff
@@ -9,18 +9,18 @@ <h3><?= e($this->property('title')) ?></h3>
             data-full-width="1"
         >
             <ul>
-            <?php if (count($rows)): ?>
```

Comment:
Please keep the original indentation

Reply:
I'll take note of your recommendation. I do not do stupid things continue to...
By the way, I'm a new boy here for one month.